### ISSUE TITLE
3pcd plan dropped? What does that mean for partitioned cookie attribute and 3pcd trial

### ISSUE THREAD

**Issue by kumarrishav** [privacysandbox/privacy-sandbox-dev-support#400]
https://privacysandbox.com/news/privacy-sandbox-update/

https://x.com/jaffathecake/status/1815475388019331200

What does that mean for partitioned cookie attribute significance and 3pcd trial (1% deprecation) and changes that were done in webview v124 related to partitioned cookie.

**Comment by dmdabbs** [comment]
Chrome announced in the past week their intent to extend the facilitated testing feature to [M129](https://chromestatus.com/roadmap). 
https://groups.google.com/a/chromium.org/g/blink-dev/c/Xgpz_MnjL4k/m/Hv9u_D0AAQAJ

Re WebView, see Intent to [Deprecate and Relaunch: CHIPS on WebView](https://groups.google.com/a/chromium.org/g/blink-dev/c/_hHg5MASz4s/m/xWu_9uZNAgAJ), 29 May.

**Comment by kumarrishav** [comment]
I didn't understand those discussions fully. It seems till M129 (Sep 2024), current behavior will be supported.

So, The current exemption (by default, who got the origin trial) is given till 31st Aug - 3pcd, earlier it was 30th June which increased to 31st Aug (another 2 months extension) 
and 3pcd origin trial token will take care of it till 27th Dec 2024 i.e this token and exemption will take care of 3pcd for 2024. 

For next year, i.e2025 onwards - Do we need to worry about partitioned cookie? Should we or Do we need to implement partitioned cookie attribute anymore? after seeing this https://privacysandbox.com/news/privacy-sandbox-update/

CC @dmdabbs

**Comment by dmdabbs** [comment]
>For next year, i.e2025 onwards - Do we need to worry about partitioned cookie? Should we or Do we need to implement partitioned cookie attribute anymore? after seeing this https://privacysandbox.com/news/privacy-sandbox-update/
CC @dmdabbs

Can't speak to your usecase @kumarrishav not knowing specifics. I'm not a Googler, just trying to provide hopefully informative references.

**Comment by kumarrishav** [comment]
Okay. Let me rephrase, Do we need to worry about partitioned cookie or implement it anymore? Currently, google has rolled it out to 1%. is plan to do it 100% still ON?

**Comment by dmdabbs** [comment]
Partitioned cookies are, as far as I know, completely deployed to all Chrome stable release users since last year.

**Comment by kumarrishav** [comment]
> Partitioned cookies are, as far as I know, completely deployed to all Chrome stable release users since last year.

Correct. But Partitioned cookies came into existence to handle third party cookie deprecation and provide an alternate way for legit use cases. But now that third party cookie deprecation plan is dropped, is there any significance of Partitioned cookies anymore? Should we implement Partitioned attribute in our cookies?

**Comment by krgovind** [comment]
@kumarrishav Just a clarification, as per the updated plan, third-party cookies can still be blocked/restricted in Chrome based on user choice, so I would recommend still investing in your site's migration to privacy-preserving alternatives like Partitioned cookies. Partitioned cookies will be supported in both modes (clients that have 3pcs restricted, and those that don't).

**Comment by kumarrishav** [comment]
@krgovind understood. That makes sense.

But that’s not new. Right? User can block third party cookies today too. Is it like there will be a new experience where it will be more intuitive n easy for user to block third party and we can expect more third party cookie block than before?

**Comment by krgovind** [comment]
@kumarrishav This will indeed be a new user experience. I don't have any additional details to share at this point, but we will post those on https://privacysandbox.com/news/ when we have them.

**Comment by kumarrishav** [comment]
@krgovind understood. We will work on that, but it will take some time.

btw, are we going to revert that 1% rollout and do pause on further rollout? If yes, can we have the timeline, so that we don’t have to worry about 3pcd origin trial token implementation and exemption.

and just focus on partition implementation.

**Comment by kumarrishav** [comment]
> @krgovind understood. We will work on that, but it will take some time.
> 
> btw, are we going to revert that 1% rollout and do pause on further rollout (or at least give permanent exemption) ? If yes, can we have the timeline, so that we don’t have to worry about 3pcd origin trial token implementation and exemption.
> 
> and just focus on partition implementation.

@krgovind  @kevinkiklee any update on this?

**Comment by krgovind** [comment]
@kumarrishav My apologies, this got lost in my backlog. Here is the update we published a little while ago: https://developers.google.com/privacy-sandbox/blog/3pc-exceptions-update (i.e. the exemption has been extended at least until we introduce a new experience in Chrome that lets people make an informed choice.

**Comment by samdutton** [comment]
I'll close this as completed, but feel free to reopen.
